utils/scapp.h

```diff
@@ -661,6 +666,11 @@ class scApp : public ktxApp {
           "  --normalize\n"
           "               Only valid for linear textures with two or three components.\n"
           "               Normalize input normals to have a unit length.\n"
+          "  --perceptual\n"
+          "               The codec should optimize perceptual error, instead of direct\n"
+          "               RMS error. This aims to improves perceived image quality, but\n"
+          "               typically lowers the measured PSNR score. Perceptual methods are\n"
+          "               currently only available for normal maps and RGB color data.\n"
```

Comment:
Please confirm that perceptual is available for normal maps. I thought Pete said it worked badly with non-color data. Please update the  documentation it normal maps are not supported.

Reply:
This is a copy past from the the encoder documentation so I assume it works. But I have asked Pete to double check as well.

```
       -perceptual
           The codec should optimize perceptual error, instead of direct
           RMS error. This aims to improves perceived image quality, but
           typically lowers the measured PSNR score. Perceptual methods are
           currently only available for normal maps and RGB color data.
```

Reply:
Pete confirmed, the documentation is correct. He also mentioned in practice the perceptual modes is a bit variable and makes parts of images worse. This bit of the code hasn't changed since 1.0 and suggested we don't make it the default option. Which we are not.